mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt

```diff
@@ -290,8 +291,20 @@ class BackgroundTasksManager : BroadcastReceiver() {
             val constraints = Constraints.Builder()
                 .setRequiredNetworkType(NetworkType.CONNECTED)
                 .build()
+
+            // Value is stored in minutes, but we need millis to compare it
+            var repeatInterval = if (interval == 0) {
+                prefs.getString(PrefKeys.SEND_DEVICE_INFO_SCHEDULE).toInt()
+            } else {
+                interval
+            } * 60 * 1000L
+            repeatInterval = max(repeatInterval, PeriodicWorkRequest.MIN_PERIODIC_INTERVAL_MILLIS)
+            var flexInterval = (repeatInterval * 0.75).toLong()
+            flexInterval = max(flexInterval, PeriodicWorkRequest.MIN_PERIODIC_FLEX_MILLIS)
```

Comment:
With the suggestion above, those two can become `val`s.

Reply:
Done